client/src/pages/addNewSitSpot/index.css

```diff
@@ -0,0 +1,110 @@
+.add-place__wrapper {
+  padding: 0 1.5rem;
+}
+
+.add-place__header {
+  background-image: url('../../assets/images/home-background.jpg');
```

Comment:
try to use another image for the background to let the header be clear

Reply:
images can be changed later, all links in the project will be redone next week